.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: python
+matrix:
+  allow_failures:
+    - python: pypy
+    - python: 3.7-dev
 python:
   - "2.6"
   - "2.7"
   - "3.4"
+  - "3.5"
+  - "3.6"
+  - "3.7-dev"
```

Comment:
Having a dev version here would require a bit of engineering resource to keep an eye on it. Are you sure you'd like to follow up on this and get this updated once 3.7 is officially released?